packages/ember-runtime/lib/mixins/target_action_support.js

```diff
@@ -9,7 +9,19 @@ import {
   Mixin,
   computed
 } from 'ember-metal';
-import { assert } from 'ember-debug';
+import { assert, deprecate } from 'ember-debug';
+
+function depricateTargetObject() {
+  deprecate(
+    'Using `targetObject` has been deprecated in TargetActionSupport Mixin',
+    false,
+    {
+      id: 'ember-runtime.mixins.target-action-support.targetObject',
+      until: '3.0.0'
```

Comment:
do we add `since` now for the svelte work, or is that information stored elsewhere?